src/_partials/_head.erb

```diff
@@ -3,6 +3,7 @@
 <%= seo %>
 <link rel="shortcut icon" type="image/svg+xml" href="<%= relative_url site.config["metadata"]["favicon_light"] %>" media="(prefers-color-scheme: light)">
 <link rel="shortcut icon" type="image/svg+xml" href="<%= relative_url site.config["metadata"]["favicon_dark"] %>" media="(prefers-color-scheme: dark)">
+<link rel="preload" href="https://bump.sh/website/_bridgetown/static/saans-uprights-variable-subset-EIB2O6P5.woff2" as="font" type="font/woff2" crossorigin fetchpriority="high">
```

Comment:
I decided to use our main website font to benefit from the fact that it may be already loaded. This can totally be discussed.